### PR TITLE
mobx integration and text only card

### PR DESCRIPTION
**Pull request by AkashGutha**
giphy-downsized

In this, i've integrated mobx with the template system.
also added a simple JSON file that represents a collection of articles

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+  '{{repeat(13)}}',
+  {
+ articleId: '{{guid()}}',
+  author: '{{firstName()}} {{surname()}}',
+  category: '{{random("typescript", "javascript", "react")}}',
+  tags: [
+      '{{repeat(3)}}',
+      '{{random("typescript", "javascript", "react")}}'
+    ],
+  isDraft: false,
```
Comment (joelhooks) [inline README.md:17]: with our api everything is "snake" case, so this would be `is_draft` for example.

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,19 @@
 import React, { Component } from "react";
-// Import routing components
 import { Route } from "react-router";
-// Import pages
 import LandingPage from "./Pages/LandingPage";
+import Store from "./Stores/ArticelsStore";
```
Comment (joelhooks) [inline src/App.js:4]: Articles has a typo

**Review by joelhooks — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AkashGutha) [reply README.md:17]: I'll fix them in the next push

File: src/App.js (hunk 2 above)
Comment (AkashGutha) [reply src/App.js:4]: I'll fix it in the next push

**Pull request merged**